static/app/views/starfish/queries/useIndexedSpans.tsx

```diff
@@ -53,7 +53,14 @@ function getEventView(
   const search = new MutableSearch([]);
 
   for (const filterName in filters) {
-    search.addFilterValue(filterName, filters[filterName]);
+    const filter = filters[filterName];
+    if (Array.isArray(filter)) {
+      filter.forEach(value => {
+        search.addFilterValue(filterName, value);
+      });
```

Comment:
```suggestion
      search.addFilterValues(filterName, filter);
```

nit: I think this does the same thing under the hood

Reply:
ah nice, updated!